var/spack/repos/builtin/packages/py-serpent/package.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# Package automatically generated using 'pip2spack' converter
+
+
+class PySerpent(PythonPackage):
+    """
+    Serialization based on ast.literal_eval
+    """
+
+    homepage = "https://github.com/irmen/Serpent"
+    pypi = 'serpent/serpent-1.40.tar.gz'
+    maintainers = ['liuyangzhuan']
+
+    version('1.9', sha256='46311ae0e93c30c3b93e9153ffd6c76e595aaaca9d390f0cda1a750d5957f90f')
```

Comment:
Can you sort these versions from newest to oldest? Also, have you actually tested all of these versions? If not, I would just add the latest version. The older versions likely contain bugs or missing features.

Reply:
I will only keep the latest 1.40